tests/arch/common/ramfunc/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -4,5 +4,5 @@
  */
 
 &cpuapp_ram0x_region {
-	perm-execute;
+	nordic,access = <NRF_OWNER_ID_APPLICATION NRF_PERM(READ, WRITE, EXEC, SECURE)>;
```

Comment:
suggestion, to avoid macrology, why don't we provide:

```c
#define NRF_PERM_R x
#define NRF_PERM_W y
...
#define NRF_PERM_RW NRF_PERM_R | NRF_PERM_W
...
```

so DT would be even less verbose, and, no macrology is involved.

Reply:
Sure. This would produce 31 definitions in total, but I don't mind it if it can be accepted.

Reply:
I, personally, prefer that, it's one-time definition work, also autocomplete will work out of the box.

Reply:
Applied.